Dockerfile

```diff
@@ -7,13 +7,13 @@
 ##########
 # USAGE:
 # To build, run this command in the directory containing this Dockerfile:
-#   > docker build . -t <imagetag>
+#   > docker build -t <imagetag> .
```

Comment:
Is this the result of a merge from master?

Reply:
I remember changing this line from original you had, because it didn't start. I will check tmrw.